packages/@react-spectrum/s2/stories/TreeView.stories.tsx

```diff
@@ -66,6 +68,16 @@ const meta: Meta<typeof TreeView> = {
         }
       },
       table: {category: 'Events'}
+    },
+    isDetached: {
```

Comment:
?

Reply:
oh, got a bit confused here, the comments on those props made it sound like they weren't fully implemented yet but I see that its more that a combination of the two + selection isn't working. I'll add descriptions and add them back in